lib/cronjobs/gcltocpu.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (config) => {
+  config.cronjobs.gcltocpu = [10, async () => {
+    const { env, db } = config.common.storage
+    if (!db) return
+    const gclToCPU = config.utils.gclToCPU || false
```

Comment:
Nitpick, but undefined is also falsy :stuck_out_tongue:

Reply:
fixed